BitFaster.Caching/Duration.cs

```diff
@@ -28,6 +28,10 @@ namespace BitFaster.Caching
 
         internal static readonly ulong epsilon = (ulong)Duration.FromMilliseconds(20).raw;
 
+#if NETCOREAPP3_0_OR_GREATER

```

Comment:
Failing test:

TLruTickCount64PolicyTests.CreateItemInitializesTimestampToNow - this test depends on epsilon, which is also a static readonly.